app/assets/javascripts/covid19/src/webapp/components/structures-table/cells/PdbCell.tsx

```diff
@@ -38,54 +99,60 @@ const PdbCell2: React.FC<{ structure: Structure; pdb: Pdb }> = React.memo(props
         </React.Fragment>
     );
 
+    const validationsTooltip = (
+        <React.Fragment>
+            {pdbValidations.map(pdbValidation => getValidation(pdbValidation))}
+        </React.Fragment>
+    );
+
+    const handleTooltipClose = React.useCallback(() => {
+        setOpen(false);
+    }, []);
+
+    const handleTooltipOpen = React.useCallback(() => {
+        setOpen(true);
+    }, []);
+
     return (
         <React.Fragment>
-            {pdb ? <Thumbnail type="pdb" value={pdb} tooltip={tooltip} /> : null}
+            {pdb ? <Thumbnail type="pdb" value={pdb} tooltip={propertiesTooltip} /> : null}
 
             {!_.isEmpty(pdbValidations) ? (
-                <div>
-                    {pdbValidations.map(pdbValidation => {
-                        switch (pdbValidation.type) {
-                            case "pdbRedo":
-                                return (
-                                    <PdbRedoBadges key="pdb-redo">
-                                        <BadgeLink
-                                            key="pdb-redo-external"
-                                            url={pdbValidation.externalLink}
-                                            text={i18n.t("PDB-Redo")}
-                                            icon="external"
-                                            color={pdbValidation.badgeColor}
-                                        />
-
-                                        <BadgeLink
-                                            key="pdb-redo-viewer"
-                                            url={pdbValidation.queryLink}
-                                            icon="viewer"
-                                            color={pdbValidation.badgeColor}
-                                        />
-                                    </PdbRedoBadges>
-                                );
-                            case "isolde":
-                                return (
-                                    <BadgeLink
-                                        key="pdb-isolde"
-                                        url={pdbValidation.queryLink}
-                                        text={i18n.t("Isolde")}
-                                        icon="viewer"
-                                        color={pdbValidation.badgeColor}
-                                    />
-                                );
-                            default:
-                                throw new Error("Unsupported");
-                        }
-                    })}
-                </div>
+                pdbValidations.length == 1 ? (
+                    getValidation(pdbValidations[0])
+                ) : (
```

Comment:
Refactor. Checking length does not help TS on the existence of pdbValidations[0]